angular/app.module.js

```diff
@@ -0,0 +1,8 @@
+var AppModule = ng.core.NgModule({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5238370)